Command/GenerateControllerCommand.php

```diff
@@ -333,4 +346,15 @@ protected function createGenerator()
     {
         return new ControllerGenerator($this->getContainer()->get('filesystem'));
     }
+
+    private function tryUpdateAnnotationRouting($bundleName, $controller)
```

Comment:
Maybe this should go in `Sensio\Bundle\GeneratorBundle\Generator\ControllerGenerator::generateRouting()` ?